components/backends/trtllm/src/dynamo/drafter/main.py

```diff
@@ -0,0 +1,165 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import asyncio
+import logging
+import signal
+import sys
+
+import uvloop
+from tensorrt_llm import SamplingParams
+from tensorrt_llm.llmapi.llm_utils import update_llm_args_with_extra_options
```

Comment:
As this is related to trtllm, it even makes more sense to not create a separate drafter module.